jest.config.js

```diff
@@ -21,8 +21,6 @@ module.exports = {
     '<rootDir>/desktop/core/src/desktop/js/parse/sql/phoenix/test',
     '<rootDir>/desktop/core/src/desktop/js/parse/sql/druid/test',
     '<rootDir>/desktop/core/src/desktop/js/parse/sql/presto/test',
-    '<rootDir>/desktop/core/src/desktop/js/parse/sql/calcite/test',
```

Comment:
@JohanAhlen  I have removed the jest.config for calcite and Flink for it to run

Reply:
It's there to reduce the CI build time. For now keep it as ignored in this list, we'll need to to split tests per jison file instead to get rid of the duplication.